pkg/reconciler.v1/common/gang_scheduler_framework.go

```diff
@@ -164,6 +165,10 @@ func (r *SchedulerFrameworkReconciler) DecoratePodForGangScheduling(
 	podTemplate *corev1.PodTemplateSpec,
 	job client.Object,
 ) {
+	if podTemplate.Spec.SchedulerName == "" && r.GetGangSchedulerName() != "default-scheduler" {
```

Comment:
```suggestion
	if podTemplate.Spec.SchedulerName == "" {
```

Let's use a logic the same as the podgroup_control.